config/prometheus/alerts.yaml

```
@@ -10,8 +10,8 @@ spec:
     rules:
     - alert: ElasticsearchOperatorRemovalWarning
       annotations:
-        message: "Elasticsearch Operator is deprecated and will be removed from the catalog on November 2025."
-        summary: "In OCP 4.16 and forward, Elasticsearch is only supported for ServiceMesh, Tracing and Kiali, and this operator will be removed from the OpenShift Operator Catalog in November 2025. Elasticsearch is no longer supported for log storage. Kibana is no longer supported in OCP 4.16 and forward."
+        message: "Elasticsearch Operator will reach end of life on November 2025."
+        summary: "IN OCP 4.16 AND FORWARD ELASTICSEARCH and KIBANA ARE NO LONGER SUPPORTED FOR LOG STORAGE AND ELASTICSEARCH IS ONLY SUPPORTED FOR SERVICEMESH, TRACING, AND KIALI"
```

Small suggestion for the alert text:

```suggestion
        summary: "ElasticSearch and Kibana are unsupported on OCP 4.16 and above when used for OpenShift Logging. Additionally, ElasticSearch Operator will stop to be supported for all use-cases in November 2025."
```

@JoaoBraveCoding @r2d2rnd Which do you prefer? Text on the PR or suggestion? Or a third option?

---

I think I prefer the one in your suggestion

---

I prefer text also